### DISCUSSION TITLE
Report Generation

### DISCUSSION THREAD

**Discussion by rupeshpyeole** [Sunbird-Obsrv/Community#129] [Issues]
Hi Team , 

We want know which job is responsible to generate **"Daily ETB Usage"** admin dashboard report.

Thank, 
Rupesh

**Comment by rupeshpyeole** [comment]
Issue description:
1.We want know Is it Hawkeye report or customized report?
2. if it is customized report then which job is responsible for it?
3. and where can we get git hub code link?

**Reply by manjudr** [reply]
@rupeshpyeole 

It's  Hawkeye report and it internally uses the DruidQueryProcessing Job and you can find the source code of the druid query processing job here - https://github.com/Sunbird-Obsrv/sunbird-core-dataproducts/blob/release-5.1.3/batch-models/src/main/scala/org/ekstep/analytics/model/DruidQueryProcessingModel.scala

**Reply by rupeshpyeole** [reply]
Hi @manjudr Thank you so much for proving us input.

**Reply by manjudr** [reply]
@rupeshpyeole  - If the issue has been resolved, please close it.
